deployment/openshift-play-scala-grpc-example.yml

```diff
@@ -0,0 +1,87 @@
+---
+apiVersion: "apps/v1beta2"
+kind: Deployment
+metadata:
+  name: "play-scala-grpc-example-v1-0-snapshot"
+  labels:
+    appName: "play-scala-grpc-example"
+    appNameVersion: "play-scala-grpc-example-v1-0-snapshot"
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      appNameVersion: "play-scala-grpc-example-v1-0-snapshot"
+  template:
+    metadata:
+      labels:
+        appName: "play-scala-grpc-example"
+        appNameVersion: "play-scala-grpc-example-v1-0-snapshot"
+    spec:
+      restartPolicy: Always
+      containers:
+        - name: "play-scala-grpc-example"
+          image: "docker-registry-default.centralpark.lightbend.com/play-scala-grpc-example/play-scala-grpc-example:1.0-SNAPSHOT"
+          imagePullPolicy: IfNotPresent
+          env:
+            - name: "JAVA_OPTS"
+              value: "-Dplay.http.secret.key=a-very-strong-key-for-production -Dplay.filters.hosts.allowed.0=myservice.example.org -Dplay.server.pidfile.path=/dev/null"
+            - name: "TRANSPORT_HTTP_BIND_ADDRESS"
+              valueFrom:
+                fieldRef:
+                  fieldPath: "status.podIP"
+            - name: "TRANSPORT_HTTPS_BIND_ADDRESS"
+              valueFrom:
+                fieldRef:
+                  fieldPath: "status.podIP"
+            - name: "DEPLOYMENT_SERVICE_NAME"
+              value: "play-scala-grpc-example"
+          ports:
+            - containerPort: 9000
+              name: http
+            - containerPort: 9443
+              name: https
+          volumeMounts: []
+          command:
+            - "/opt/docker/bin/play-scala-grpc-example"
+      volumes: []
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    appName: "play-scala-grpc-example"
+  name: "play-scala-grpc-example"
+spec:
+  ports:
+    - name: http
+      port: 9000
+      protocol: TCP
+    - name: https
+      port: 9443
+      protocol: TCP
+  selector:
+    appName: "play-scala-grpc-example"
+---
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  labels:
+    appName: play-scala-grpc-example
+  name: play-scala-grpc-route
+  namespace: play-scala-grpc-example
+  selfLink: >-
+    /apis/route.openshift.io/v1/namespaces/play-scala-grpc-example/routes/play-scala-grpc-route
+spec:
+  host: myservice.example.org
+  port:
+    targetPort: http
+  to:
+    kind: Service
+    name: play-scala-grpc-example
+    weight: 100
+  wildcardPolicy: None
+status:
+  ingress:
+  - conditions:
+    host: myservice.example.org
+    routerName: router
```

Comment:
This YAML is a stripped down version of the YAML OpenShift generates when creating a `Route` from the GUI. I've removed a few fields that I thought irrelevant. It's possible this YAML can be reduced a bit more. I don't think we should block this PR on that potential improvement.